src/m365/spo/commands/user/user-remove.spec.ts

```diff
@@ -119,6 +192,57 @@ describe(commands.USER_REMOVE, () => {
     assert.notStrictEqual(actual, true);
   });
 
+
+  it('fails validation if entraGroupId is not a valid id', async () => {
+    const actual = await command.validate({ options: { webUrl: validWebUrl, entraGroupId: 'invalid' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation if id is not a valid id', async () => {
+    const actual = await command.validate({ options: { webUrl: validWebUrl, id: 'invalid' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation if userName is not a valid user principal name', async () => {
+    const actual = await command.validate({ options: { webUrl: validWebUrl, userName: 'invalid' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation if email is not a valid user principal name', async () => {
+    const actual = await command.validate({ options: { webUrl: validWebUrl, email: 'invalid' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+  it('passes validation url is valid and id is passed', async () => {
```

Comment:
let's add a bit of space here to make it more readable and align the formating with rest of the file 

```suggestion
  });
  
  it('passes validation url is valid and id is passed', async () => {
```